python_modules/dagster/dagster/_serdes/serdes.py

```diff
@@ -424,7 +424,7 @@ def is_whitelisted_for_serdes_object(
     if (
         (isinstance(val, tuple) and hasattr(val, "_fields"))  # NamedTuple
         or (is_dataclass(val) and not isinstance(val, type))  # dataclass instance
-        or isinstance(val, pydantic.BaseModel)
```

Comment:
the top level pydantic module has this machinery
https://github.com/pydantic/pydantic/blob/main/pydantic/__init__.py#L383-L396
which is expensive to go through in a hot code path

importing specific attributes pays that cost once at import time instead